libs/core/src/lib/carousel/carousel.component.ts

```diff
@@ -557,8 +546,10 @@ export class CarouselComponent implements OnInit, AfterContentInit, AfterViewIni
     private _subscribeToRtl(): void {
         const refreshDirection = () => {
             this.dir = this._rtlService?.rtl.getValue() ? 'rtl' : 'ltr';
```

Comment:
can be simplified to the getter like that

```
get dir(): string {
  return this._isRtl() ? 'rtl' : 'ltr';
}
```

Reply:
fixed